test/functional/test_unversioned_shebangs.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+
+from taskotron_python_versions.unversioned_shebangs import (
+    matches,
+    get_problematic_files,
+    get_scripts_summary,
+)
+from taskotron_python_versions.common import Package
+
+fixtures_dir = os.path.dirname(
```

Comment:
See `pkg` and `gpkg` functions from common and use them instead please. That would amek your code more consistent with other tests.

They work like this:

```python
gpkg('tracer*')  # -> Package('../fixtures/tracer-0.6.9-2.fc25.noarch.rpm')
gpkg('tracer*').path  # -> '../fixtures/tracer-0.6.9-2.fc25.noarch.rpm'
pkg('tracer-0.6.9-2.fc25.noarch.rpm') # -> '../fixtures/tracer-0.6.9-2.fc25.noarch.rpm'
```

(Untested, but at least they work similarly to this.)